### PR TITLE
Added team for etcd-io/auger

### PR DESCRIPTION
**Pull request by jmhbnz**
Fixes https://github.com/kubernetes/org/issues/4754

/hold

Hold pending repo migration.

### CODE REVIEW

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
To use the repo permissions features the repo needs to be added to the list here:

https://github.com/kubernetes/org/blob/25da6e3d0bbc154da53d998899dc8aa388d9eb33/config/restrictions.yaml#L376-L389

right now they're all pretty much just explicitly defined, but it should work for regex patterns.

**Comment by jmhbnz** [comment]
Good catch @mrbobbytables - Should be sorted now 👍🏻

**Comment by jmhbnz** [comment]
/unhold

Repo migration has now been completed, https://github.com/etcd-io/auger is live.

File: config/etcd-io/org.yaml [hunk 1]
```diff
@@ -121,6 +121,15 @@ teams:
...
       website: admin
+  maintainers-auger:
+    description: Granted write access to auger
+    members:
+    - jmhbnz
+    - siyuanfoundation
+    - wenjiaswe
+    privacy: closed
+    repos:
+      auger: admin
```
Comment (MadhavJivrajani) [inline config/etcd-io/org.yaml:132]: Just making sure you meant `admin` here - the use of `admin` vs `maintain` is not consistent across etcd teams (might be worth changing that in the future!)

**Review by MadhavJivrajani — COMMENTED** [review]
Just one comment, otherwise lgtm

File: config/etcd-io/org.yaml (hunk 1 above)
Comment (MadhavJivrajani) [reply config/etcd-io/org.yaml:132]: If you did mean admin, I would also recommend reflecting that in the team description by s/write/admin

Comment (jmhbnz) [reply config/etcd-io/org.yaml]: Good catch, I'll confess I had just copied it from the above entry which said `admin`.  I think `maintain` should be fine as that seems to be used elsewhere.  We can probably change website to `maintain` but I would prefer to do that separately.

**Review by MadhavJivrajani — APPROVED** [review]
/lgtm
/approve

Thanks!

**Pull request merged**